src/css/sidebar.css

```diff
@@ -199,103 +168,194 @@
 }
 
 #side-bar .heading p,
-#side-bar .side-block > .collapsible-block .collapsible-block-link {
+#side-bar .collapsible-block .collapsible-block-link {
 	--text-shadow: rgb(var(--swatch-border-color, 12, 12, 12));
 	--wght: 700;
+	display: flex;
 	letter-spacing: 0.2rem;
 	color: rgb(252, 252, 252);
 	color: rgb(var(--swatch-text-secondary-color));
 	height: 1.25rem;
+	margin: 0;
 	line-height: 1.25rem;
 	text-transform: uppercase;
 	font-size: calc(0.875rem * 0.9);
 	font-size: calc((var(--base-font-size) * (14/15)) * 0.9);
 	font-weight: 700;
 	letter-spacing: 0.016em;
-	text-align: center;
+	justify-content: center;
+	align-items: center;
 	text-shadow: 0.063rem 0.063rem 0.063rem rgb(12, 12, 12);
 	text-shadow: 0.063rem 0.063rem 0.063rem var(--text-shadow);
-	display: block;
 }
 
 #side-bar .heading p {
 	cursor: default;
 }
 
-#side-bar div.menu-item {
-	margin: 0;
-	text-align: right;
-	display: flex;
-	flex-direction: row;
-	flex-wrap: nowrap;
-	align-items: center;
-	justify-content: flex-start;
-	font-size: 0;
-	min-height: calc(1.5rem - 0.125rem);
-	min-height: calc(1.5rem - var(--sidebar-internal-border-thickness));
+/* Collapsibles in the sidebar */
+
+#side-bar div.collapsible-block {
+	width: 100%;
+	flex-basis: 100%;
+}
+
+#side-bar div.collapsible-block a.collapsible-block-link,
+#side-bar .side-block>.collapsible-block a.collapsible-block-link {
+	padding-right: 2rem;
+	position: relative;
+}
+
+#side-bar a.collapsible-block-link:hover {
+	text-decoration: none;
+	background-color: rgba(180, 35, 35);
+	background-color: rgba(var(--swatch-primary));
+}
+
+/* create a dropdown triangle next to the collapsible */
+#side-bar div.menu-item a.collapsible-block-link::after,
+#side-bar .side-block>.collapsible-block a.collapsible-block-link::after {
+	content: "";
+	position: absolute;
+	height: 100%;
+	width: 2rem;
+	top: 0;
+	right: 0;
+	background-image: url(http://scptestwiki.wdfiles.com/local--files/squeeze%3Anav%3Aside/arr_drop.svg);
```

Comment:
HTTP link, also not pointing to scp-wiki

Reply:
Got it! Thanks for the catch! I swapped it to an embedded svg instead.